mlrun/utils/notifications/notification/slack.py

```diff
@@ -144,10 +144,19 @@ def _get_alert_fields(
             data_text = "\n".join(data_lines)
             line.append(self._get_slack_row(f"*Event data:*\n{data_text}"))
 
-        if url := mlrun.utils.helpers.get_ui_url(
-            alert.project, event_data.entity.ids[0]
-        ):
-            line.append(self._get_slack_row(f"*Overview:*\n<{url}|*Job overview*>"))
+        uid = event_data.value_dict.get("uid")
+        if uid:  # JOB entity
```

Comment:
You should use `event_data.entity.kind` to decide what type of entity it is, don't rely on value dict fields which are not mandatory and can change.